ui/app/src/utils/itemActions.ts

```diff
@@ -654,9 +659,13 @@ export const itemActionDispatcher = ({
               showCutItemSuccessNotification()
             ]);
 
-            if (dependant?.length) {
-              const items = parseLegacyItemToSandBoxItem(dependant);
-              dispatch(showBrokenReferencesDialog({ path, references: items, onContinue: actionToDispatch }));
+            if (dependantItems?.length) {
+              fetchItemsByPath(
+                site,
+                dependantItems.map((item) => item.uri ?? item.path)
```

Comment:
@jvega190 Is this an api1? Why the `item.uri` possibility? Please check if there's an API2 and create a ticket for refactor if possible.